### ISSUE TITLE
Why just discard text encoder  and representation enhancement module?

### ISSUE THREAD

**Issue by XiaogangYin** [valley-vl/FashionLOGO#1]
If I want a ultimate performace, it is better to reserve the text encoder  and representation enhancement module, isn't it?

I do not see the discussion in the ablation studies.

Thanks

**Comment by valley-vl** [comment]
@XiaogangYin No, it is no need to reserve the text encoder to get an ultimate performance. Our model aims to utilize multi-modal solutions for logo embedding training where image embedding queries learn supplementary knowledge from textual embeddings automatically. We believe such a training method results in a powerful image encoder which is different from and superior than traditional method. Also, we remove text encoder in inference stage to ensure fairness under comparison with traditional methods and ensure efficiency to make it a practical model that can be used directly. We believe it is one of the advantages of our work that removing the text encoder branch in the inference stage.
